<?php
$this->pageTitle=Yii::app()->name . ' - Glossary';
?>

<div class=WordSection1 style='layout-grid:18.0pt'>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-size:20.0pt;font-family:"Times New Roman","serif";
color:#595959'>Glossary</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><u><span
lang=EN-US style='font-size:15.0pt;font-family:"Helvetica","sans-serif";
color:#595959'>Annotation and Chromosome</span></u></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
layout-grid-mode:char'><b><span lang=EN-US style='font-size:5.0pt;font-family:
"Helvetica","sans-serif";color:#595959'>&nbsp;</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Evidence
level:</span></b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> There are currently 2 levels of evidence for novel lncRNAs. </span></p>

<p class=MsoNormal style='margin-left:12.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:24.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Level 1: Supported by either one RNA-Seq study and tiling array
study</span></p>

<p class=MsoNormal style='margin-left:12.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:24.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Level 2: Supported by both RNA-Seq studies and tiling array
study</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Type:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>
According to the relative position to coding gene (annotated by Ref-Seq), all
lncRNAs are classified to at least one of the following types.</span></p>

<p class=MsoListParagraph style='margin-left:48.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-24.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>a)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Exonic:
overlapping with any exon of a coding gene;</span></p>

<p class=MsoListParagraph style='margin-left:48.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-24.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Intronic:
within a coding gene, but not overlap with any exon of that coding gene;</span></p>

<p class=MsoListParagraph style='margin-left:48.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-24.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Promoter-associated:
overlapping with promoter (2000-bp upstream) of a coding gene, but not with any
exon of that coding gene; and/or</span></p>

<p class=MsoListParagraph style='margin-left:48.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-24.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>d)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Intergenic:
not overlapping with any coding gene or its promoter.</span></p>

<p class=MsoNormal style='margin-left:24.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:24.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Note that more than one 'type' will be assigned to a lncRNA relative
to different coding genes. For example, some lncRNAs are classified as 'Promoter-associated
and exonic', indicating that they are promoter-associated relative to one
coding gene, and exonic to another.</span></p>

<p class=MsoNormal style='margin-left:24.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:24.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><u><span
lang=EN-US style='font-size:15.0pt;font-family:"Helvetica","sans-serif";
color:#595959'>Regulation</span></u></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
layout-grid-mode:char'><span lang=EN-US style='font-size:5.0pt;font-family:
"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Poly A
(testis):</span></b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> Presence of Polyadenylation signal within -50 to +200 bp
relative to 3'-end in testis tissue. Poly-A signal is regarded as a 3' evidence
for a transcript.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Poly A-sequencing from Merck Research
Laboratories, mm9)</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Poly A
(other tissues):</span></b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> Presence of Polyadenylation signal within -50 to +200 bp
relative to 3'-end in either brain, kidney, liver or muscle tissue. Poly-A
signal is regarded as a 3' evidence for a transcript.</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Poly A-sequencing from Merck Research
Laboratories, mm9)</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>DHS:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
(overlapping for &#8805;10 bp) of strong (top 50% by intensity of peaks) DNaseI
hypersensitive sites (DHS) signals in promoter (2000 bp upstream to
transcription starting site) in either B-cells, cerebellum, cerebrum, fat pad,
fibroblast, heart, kidney, large intestine, liver, lung, retina, skeletal
muscle, spleen, T cells, thymus, or whole brain samples from 8-week-old mouse.
DHS is regarded as a signal for promoter.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: DNaseI Hypersensitivity by Digital DNaseI
from ENCODE/ University of Washington) </span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>CAGE:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of Riken Cap analysis of gene expression (CAGE) signal within -200 to +50 bp
relative to 5'-end. CAGE signal is regarded as a 5' evidence for a transcript.</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Riken CAGE - Associated Transcript
Clusters, lifted from mm7)</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>H3K4me1:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of strong signal of histone 3 mono-methylated lysine 4 in promoter or gene body
in testis tissue. This is a histone signal for enhancer.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Histone Mods by ChIP-seq from
ENCODE/LICR)</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>6.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>H3K4me3:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of strong signal of histone 3 tri-methylated lysine 4 in promoter in testis
tissue. This is a histone signal for active promoter.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Histone Mods by ChIP-seq from
ENCODE/LICR)</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>7.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>H3K27ac:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of strong signal of histone 3 acetylated lysine 27 in promoter in testis
tissue. This is a histone signal for active promoter.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Histone Mods by ChIP-seq from
ENCODE/LICR)</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>8.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>H3K27me3:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of strong signal of histone 3 tri-methylated lysine 27 in promoter in testis
tissue. This is a histone signal for inactive promoter.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Histone Mods by ChIP-seq from
ENCODE/LICR)</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>9.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>H3K36me3:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Presence
of strong signal of histone 3 tri-methylated lysine 36 in promoter or gene body
in testis tissue. This is a histone signal for transcription.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: Histone Mods by ChIP-seq from
ENCODE/LICR)</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>10.<span
style='font:7.0pt "Times New Roman"'> </span></span><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>Conserved element:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> The
lncRNA loci overlap with PhastCons Placental Mammal Conserved Elements, 30-way
Multiz Alignment. This suggests the lncRNA is conserved among mammals.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>(Reference UCSC track: 30-Way Multiz Alignment &amp;
Conservation)</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>11.<span
style='font:7.0pt "Times New Roman"'> </span></span><b><i><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>CIS</span></i></b><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>
regulation:</span></b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> Neighbour coding genes, both up- or down-stream to the lncRNA,
and their gene ontology, if any.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>12.<span
style='font:7.0pt "Times New Roman"'> </span></span><b><i><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>TRANS </span></i></b><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>regulation:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Coding
genes with &#8805;85% sequence homology within a region of &#8805;200 bp to a lncRNA will
be listed out, together with their gene ontology, if any.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><u><span
lang=EN-US style='font-size:16.0pt;font-family:"Helvetica","sans-serif";
color:#595959'>Expression data</span></u></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>SAGE:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'> Counts
of tag that are mapped to each lncRNA from serial analysis of gene expression
experiment. The total numbers of tag counts in the three stages are 150,793 in
type A spermatogonia, 151,181 in pachytene spermatocytes, and 150,144 in round
spermatids. Tag counts &#8805;5 are commonly regarded as abundant.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>RNA-Seq:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>
Expression in RPKM for each lncRNA in 6-day spermatogonia, pachytene
spermatocytes and round spermatids. Expression&#8805;3 RPKM are commonly regarded as
abundant.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Microarray:</span></b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>
Normalized intensity (in log2 scale) of samples from neonatal (6-day) and adult
(8-week) whole testis transcriptome.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Stage
specific:</span></b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> LncRNAs with expression &#8805;3 RPKM and fold-change &#8805;3 higher than
that at the other two stages will be selected.</span></p>

</div>