<?php
$this->pageTitle=Yii::app()->name . ' - Help';
?>
<style>p.MsoNormal a{text-decoration: none;}</style>

<div class=WordSection1 style='layout-grid:18.0pt'>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>1.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>What is GermlncRNA
Database?</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>GermlncRNA Database is the first catalog of annotated and novel
lncRNAs in mouse spermatogenesis, with expression level and associated
regulatory features. Details on genomic distributions and features can be found
in <b>Statistics<i> </i></b>section.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>2.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>Why spermatogenesis?</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Spermatogenesis is a complex developmental process whereby
undifferentiated spermatogonia are differentiated into functional sperms
through a series of tightly regulated proliferation and differentiation programs.
As a result, identifying the critical genes controlling these processes will
not only dissect the basic molecular mechanisms, but also allow identification
of therapeutic targets for cancer, reproductive and regenerative medicine.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Despite the biological importance of male germ cell development,
the underlying mechanisms of the stage-specific gene regulation and cellular
transition during spermatogenesis remain largely unknown. Previous genomic
studies on transcriptome profiling were limited to protein-coding genes. It is
now clear that protein-coding genes only represent small percentage of
transcriptome; most are non-coding transcripts. The functional importance of
this ¡§dark matter¡š in germ cell transcriptome remains largely elusive.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph align=center style='margin-left:0cm;text-align:center'><span
lang=EN-US style='font-family:"Helvetica","sans-serif"'><img width=646
height=194 id="Picture 8" src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image001.jpg"></span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>3.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>Why are we
interested in long noncoding RNA?</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Long noncoding RNA (lncRNA) is a group of untranslated
transcripts. It had once been considered as junk transcripts, but recently
demonstrated to be a novel and versatile layer of molecular regulation. However
the function of majority of them still remained mysterious. A few
spermatogenesis-related lncRNA candidates were characterized in recent reports,
but the number is far away from completeness. Thanks to recent advance in
high-throughput transcriptomic platforms, including next generation RNA
sequencing (RNA-Seq), microarray and serial analysis of gene expression (SAGE),
a large number of lncRNAs have been discovered. It is a good opportunity now to
investigate this novel biomolecules </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_8" title="Luk, 2014 #32821"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Luk, et al., 2014</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>. We hope
GermlncRNA Database can further enhance our understanding on lncRNA biology.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>4.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>Why did we identify
the novel lncRNAs in GermlncRNA Database?</span></b></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Although there are thousands of lncRNA transcripts previously
annotated using high-throughput transcriptomic techniques, the current
annotations are built on studies using other model cell types, but not cell
types in spermatogenesis <i>per se</i>. Given lncRNAs are expressed in a rather
cell type-specific manner, over a thousand of novel lncRNAs not annotated by
current databases are likely to be spermatogenesis-specific and may be carrying
important biological functions. </span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>5.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>How were novel
lncRNAs in GermlncRNA Database discovered?</span></b></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>To identify germ cell specific lncRNAs that may be missed by
public annotations, we developed a<b> </b><i>de novo</i> lncRNA discovery
pipeline known as <b>Hybrid Transcriptome Assembly (HTA)</b> approach<b>, </b>which
assemble male germ cell transcriptome data from different expression platforms,
including RNA-Seq and SAGE/tiling microarray data we published previously:</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#404040'>&nbsp;</span><span
lang=EN-US><img border=0 width=269 height=252 src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image002.png"></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>HTA bioinformatics pipeline:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Helvetica","sans-serif"'><img border=0 width=725
height=501 id="Picture 7" src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image003.jpg"></span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>We identified &gt;2000 unannotated intergenic lncRNAs and term
them 'novel lncRNAs'. Their GermlncRNA identities have a suffix -d to indicate
they are discovered by the <i>de novo</i> pipeline.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>6.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>Why is the advantage
of HTA?</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>HTA not only allows
identification of novel lncRNAs but also improves the lncRNA identification
reliability. By comparing with conventional annotation pipeline, we observed
improvement on specificity by more than 2-fold without sacrificing sensitivity</span></p>

<p class=MsoListParagraph align=center style='margin-left:18.0pt;text-align:
center'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>.</span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'> <img border=0 width=529 height=295 id="Chart 1"
src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image004.gif"></span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#1F497D'>&nbsp;</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>7.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>What is the evidence
level of predicted novel lncRNA transcripts?</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>Novel lncRNAs identified by HTA were classified based on number
of studies that support the lncRNA.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<div align=center>

<table class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0" style="margin-left:20pt;border-collapse:collapse;border:none;width:800px;">
 <tbody><tr>
  <td width="283" valign="top" style="width:212.6pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:solid windowtext 1.0pt;border-right:none;
  padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Evidence level of </span></p>
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">predicted novel
  lncRNA</span></p>
  </td>
  <td width="447" valign="top" style="width:335.0pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:solid windowtext 1.0pt;border-right:none;
  padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Transcription
  evidence</span></p>
  </td>
 </tr>
 <tr>
  <td width="283" valign="top" style="width:212.6pt;border:none;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">1</span></p>
  </td>
  <td width="447" valign="top" style="width:335.0pt;border:none;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Either RNA-Seq
  study, plus tiling array</span></p>
  </td>
 </tr>
 <tr>
  <td width="283" valign="top" style="width:212.6pt;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">2</span></p>
  </td>
  <td width="447" valign="top" style="width:335.0pt;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Both RNA-Seq
  studies, plus tiling array</span></p>
  </td>
 </tr>
 <tr>
  <td width="283" valign="top" style="width:212.6pt;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">3</span></p>
  </td>
  <td width="447" valign="top" style="width:335.0pt;padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Either RNA-Seq
  study, plus SAGE and tiling array</span></p>
  </td>
 </tr>
 <tr>
  <td width="283" valign="top" style="width:212.6pt;border:none;border-bottom:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">4</span></p>
  </td>
  <td width="447" valign="top" style="width:335.0pt;border:none;border-bottom:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt">
  <p class="MsoNormal" align="center" style="text-align:center"><span lang="EN-US" style="font-family:&quot;Helvetica&quot;,&quot;sans-serif&quot;;color:#404040">Both RNA-Seq
  studies, plus SAGE and tiling array</span></p>
  </td>
 </tr>
</tbody></table>

</div>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>Here is a typical example of novel lncRNA transcript with
evidence level 4 presented in UCSC Genomic Browser.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
text-indent:18.0pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#404040'>&nbsp;</span></p>

<p class=MsoListParagraph align=center style='margin-left:18.0pt;text-align:
center'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'><img
border=0 width=851 height=638 id="Picture 6" src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image005.jpg"></span></p>

<p class=MsoListParagraph style='margin-left:18.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#1F497D'>&nbsp;</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>8.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>What are the
annotated lncRNAs resources employed in GermlncRNA?</span></b></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>The following reference databases were used to provide the
annotation of known lncRNA transcripts, and the coordinates are based on
NBCI37/mm9 genome assembly.</span></p>

<p class=MsoListParagraph style='margin-left:61.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.5pt'><a name="OLE_LINK1"></a><a
name="OLE_LINK2"></a><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>a)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Ensembl
65 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_3" title="Hubbard, 2002 #32818"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Hubbard,
et al., 2002</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.5pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>UCSC
Genes </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_2" title="Hsu, 2006 #32819"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Hsu, et al., 2006</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.5pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>NCBI
Reference Sequence (RefSeq) </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_9" title="Pruitt, 2007 #8999"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Pruitt,
et al., 2007</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.5pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>d)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>NONCODE
v4 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_12" title="Xie, 2014 #32820"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Xie,
et al., 2014</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.5pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>e)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>fRNAdb
v3.4 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_5" title="Kin, 2007 #922"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Kin, et al., 2007</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>9.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>What are the
transcriptome resources employed in GermlncRNA?</span></b></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>High-throughput transcriptomic datasets used to build up this
database include the followings:</span></p>

<p class=MsoListParagraph style='margin-left:61.6pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.6pt'><a name="OLE_LINK3"></a><a
name="OLE_LINK5"></a><a name="OLE_LINK4"></a><span lang=EN-US style='font-family:
"Helvetica","sans-serif";color:#595959'>a)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>GermSAGE developed by our group previously </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_6" title="Lee, 2009 #2072"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Lee, et al., 2009</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.6pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.6pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Tiling
array dataset deposited by our group under GSE55381 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_7" title="Lee, 2012 #1919"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Lee, et al., 2012</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.6pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.6pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>RNA-Seq
dataset deposited under SRP018124 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_10" title="Soumillon, 2013 #32736"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Soumillon,
et al., 2013</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.6pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.6pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>d)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>RNA-Seq
dataset deposited under SRP010262 </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_1" title="Gan, 2013 #32631"><span style='font-family:
"Helvetica","sans-serif";color:#595959;text-decoration:none'>Gan, et al., 2013</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>)</span></p>

<p class=MsoListParagraph style='margin-left:61.6pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-19.6pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>e)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>LncRNA
microarray dataset deposited under GSE43442 </span><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_11" title="Sun, 2013 #32712"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Sun,
et al., 2013</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-align:justify;text-justify:
inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-18.0pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>10.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp;&nbsp;&nbsp; </span></b><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>How to get started
in GermlncRNA Database?</span></b></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>In <b>Data Search</b> page, users can search for lncRNAs by:</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>a)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Annotation:
transcript I.D. in different databases;</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Chromosome
number;</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Annotation
status: annotated or novel;</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>d)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Potential
regulation features: PolyA tail, DNase hypersensitive sites, CAGE, 5 types of
histone modification signals, and conserved elements; and</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>e)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Expression
data: quantitative expression signals at three developmental stages (type A
spermatogonia, pachytene spermatocytes, and round spermatids) generated by SAGE
tag count, whole-genome tiling microarray, and RNA-Seq platforms. </span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>The searching results are presented in table form. Each lncRNA
entry occupies an individual row, while showing which column to provide
relevant information are subjected to user's choice in the <b>Data
Search-Select Columns </b>panel. A screenshot with brief explanations is showed
below.</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>&nbsp;</span></p>

<p class=MsoListParagraph align=center style='margin-left:18.0pt;text-align:
center'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'><img
border=0 width=770 height=753 id="Picture 3" src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image006.jpg"></span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:18.0pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:22.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-22.5pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>11.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp; </span></b><b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#1F497D'>What is the predicted <i>cis</i>- and <i>trans</i>-regulatory
information?</span></b></p>

<p class=MsoListParagraph style='margin-left:21.7pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Many characterized lncRNAs have regulatory effects&nbsp;on genes
in close proximity (in <i>cis</i>-) or with similar sequence(s)&nbsp;in other
genomic regions (in <i>trans</i>-), we tried to predict&nbsp;the possible
functions of novel lncRNAs based on these associations, as&nbsp;defined below:</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>a)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><i><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Cis</span></i><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>-association:
Alignment hit on&nbsp;neighbour coding genes, both up- or down-stream.</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><i><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Trans</span></i><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>-association:
Alignment hit with any&nbsp;coding gene(s) with &#8805;85% sequence homology
within&nbsp;at least&nbsp;200 bp.</span></p>

<p class=MsoListParagraph align=center style='margin-left:0cm;text-align:center'><span
lang=EN-US style='font-family:"Helvetica","sans-serif"'><img border=0
width=516 height=227 id="Picture 25" src="<?php echo Yii::app()->request->baseUrl; ?>/images/help_image007.jpg"></span></p>

<p class=MsoListParagraph style='margin-left:22.5pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:22.5pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-22.5pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>12.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp; </span></b><b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#1F497D'>What is the predicted microRNA target information?</span></b></p>

<p class=MsoListParagraph style='margin-left:22.5pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>Many characterized lncRNAs interact with microRNAs (miRNAs) and regulate
the miRNA regulatory pathways, or vice versa. Here we predicted the potential miRNA
targets of novel lncRNAs by PITA algorithm </span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>(</span><span
lang=EN-US><a href="#_ENREF_4" title="Kertesz, 2007 #32817"><span
style='font-family:"Helvetica","sans-serif";color:#595959;text-decoration:none'>Kertesz,
et al., 2007</span></a></span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>)</span><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>. A cutoff at combined interaction energy (&#916;&#916;G) &#8804;-10 was taken
as positive for valid interaction.</span></p>

<p class=MsoListParagraph style='margin-left:22.5pt;text-align:justify;
text-justify:inter-ideograph'><span lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoListParagraph style='margin-left:21.3pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.3pt'><b><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#1F497D'>13.</span></b><b><span
lang=EN-US style='font-size:7.0pt;font-family:"Helvetica","sans-serif";
color:#1F497D'>&nbsp; </span></b><b><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#1F497D'>How to make use of the searching results?</span></b></p>

<p class=MsoNormal style='margin-left:22.4pt;text-align:justify;text-justify:
inter-ideograph;text-indent:.2pt'><span lang=EN-US style='font-family:"Helvetica","sans-serif";
color:#595959'>For advanced analysis, users can</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>a)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Download
the whole researching results in CSV format; </span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>b)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Visualize
lncRNA transcript in UCSC Genome Browser by clicking on GlncRNA I.D. For
further information about UCSC Genome Browser, please refer to </span><span
lang=EN-US><a href="http://genome.ucsc.edu/goldenPath/help/hgTracksHelp.html"><span
style='font-family:"Helvetica","sans-serif"'>here</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>; or</span></p>

<p class=MsoListParagraph style='margin-left:63.0pt;text-align:justify;
text-justify:inter-ideograph;text-indent:-21.0pt'><span lang=EN-US
style='font-family:"Helvetica","sans-serif";color:#595959'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>Export to
Galaxy for further analysis. Please refer to Galaxy wiki (</span><span
lang=EN-US><a href="https://wiki.galaxyproject.org/"><span style='font-family:
"Helvetica","sans-serif"'>https://wiki.galaxyproject.org/</span></a></span><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>) for
Galaxy operation.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#595959'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='font-family:"Helvetica","sans-serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b><span
lang=EN-US style='font-family:"Helvetica","sans-serif";color:#1F497D'>References</span></b></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_1"><span lang=EN-US style='color:#595959'>Gan, H.<i>, et al.</i>
Dynamics of 5-hydroxymethylcytosine during mouse spermatogenesis. <i>Nature
Communications</i> 2013;4.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_2"><span lang=EN-US style='color:#595959'>Hsu, F.<i>, et al.</i>
The UCSC Known Genes. <i>Bioinformatics</i> 2006;22(9):1036-1046.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_3"><span lang=EN-US style='color:#595959'>Hubbard, T.<i>, et al.</i>
The Ensembl genome database project. <i>Nucleic Acids Res</i> 2002;30(1):38-41.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_4"><span lang=EN-US style='color:#595959'>Kertesz, M.<i>, et al.</i>
The role of site accessibility in microRNA target recognition. <i>Nat Genet</i>
2007;39(10):1278-1284.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_5"><span lang=EN-US style='color:#595959'>Kin, T.<i>, et al.</i>
fRNAdb: a platform for mining/annotating functional RNA candidates from
non-coding RNA sequences. <i>Nucleic Acids Res</i> 2007;35(Database
issue):D145-148.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_6"><span lang=EN-US style='color:#595959'>Lee, T.L.<i>, et al.</i>
GermSAGE: a comprehensive SAGE database for transcript discovery on male germ
cell development. <i>Nucleic Acids Res</i> 2009;37(Database issue):D891-897.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_7"><span lang=EN-US style='color:#595959'>Lee, T.L., Rennert, O.M.
and Chan, W.Y. Revealing the transcriptome landscape of mouse spermatogonial
cells by tiling microarray. <i>Methods Mol Biol</i> 2012;825:75-92.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_8"><span lang=EN-US style='color:#595959'>Luk, A.C.<i>, et al.</i>
Long noncoding RNAs in spermatogenesis: insights from recent high-throughput
transcriptome studies. <i>Reproduction</i> 2014;147(5):R131-141.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_9"><span lang=EN-US style='color:#595959'>Pruitt, K.D., Tatusova,
T. and Maglott, D.R. NCBI reference sequences (RefSeq): a curated non-redundant
sequence database of genomes, transcripts and proteins. <i>Nucleic Acids Res</i>
2007;35(Database issue):D61-65.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_10"><span lang=EN-US style='color:#595959'>Soumillon, M.<i>, et
al.</i> Cellular source and mechanisms of high transcriptome complexity in the
mammalian testis. <i>Cell Reports</i> 2013;3(6):2179-2190.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_11"><span lang=EN-US style='color:#595959'>Sun, J., Lin, Y. and
Wu, J. Long Non-Coding RNA Expression Profiling of Mouse Testis during
Postnatal Development. <i>PLoS One</i> 2013;8(10):e75750.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name="_ENREF_12"><span lang=EN-US style='color:#595959'>Xie, C.<i>, et al.</i>
NONCODEv4: exploring the world of long non-coding RNA genes. <i>Nucleic Acids
Res</i> 2014;42(Database issue):D98-103.</span></a></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-US style='color:#595959'>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

</div>